Content.Server/Mech/Systems/MechSystem.cs

```diff
@@ -228,6 +232,17 @@ private void OnMechEntry(EntityUid uid, MechComponent component, MechEntryEvent
             return;
         }
 
+        // Frontier - Make AI Attack mechs based on user.
+        if (TryComp<MobStateComponent>(args.User, out var state))
```

Comment:
Question about style - state is unused, should be replaced with `_`?
```suggestion
        if (TryComp<MobStateComponent>(args.User, out var _))
```